sql/mysql/migrate.go

```diff
@@ -40,7 +40,7 @@ func (p *planApply) PlanChanges(_ context.Context, name string, changes []schema
 		return nil, err
 	}
 	for _, c := range s.Changes {
-		if c.Reverse == "" {
+		if stmts, _ := c.ReverseStmts(); len(stmts) == 0 {
```

Comment:
Why ignore the error? It is not guaranteed we have `string | []string` here, is it?

Reply:
I feel comfortable ignoring the error in this case because we build the changes in this file.

Reply:
By using a `DiffHook` a user can add a `Reverse` that is not a `string` or `[]string`.